numpy/lib/function_base.py

```diff
@@ -4263,8 +4264,34 @@ def percentile(a, q, axis=None, out=None,
     >>> assert not np.all(a == b)
 
     """
-    q = array(q, dtype=np.float64, copy=True)
-    r, k = _ureduce(a, func=_percentile, q=q, axis=axis, out=out,
+    return quantile(a, np.divide(q, 100), axis, out,
+        overwrite_input, interpolation, keepdims)
+
+
+def quantile(a, q, axis=None, out=None,
+             overwrite_input=False, interpolation='linear', keepdims=False):
+    """
+    Compute the qth quantile of the data along the specified axis.
+
+    Takes exactly the same arguments as `percentile`, but with q in [0, 1],
+    rather than [0, 100]
+
```

Comment:
I don't think you should cheat on the documentation like that, just make the usual docstring. I don't see a need to mention percentile here, just having it in the `See Also` section should be enough.